apps/entity-service/src/entities/project-pitch.service.ts

```diff
@@ -36,44 +27,48 @@ export class ProjectPitchService {
       builder.pageNumber(pageNumber);
     }
 
-    if (params.search) {
+    if (query.search) {
       builder.where({
         [Op.or]: [
-          { projectName: { [Op.like]: `%${params.search}%` } },
-          { "$organisation.name$": { [Op.like]: `%${params.search}%` } }
+          { projectName: { [Op.like]: `%${query.search}%` } },
+          { "$organisation.name$": { [Op.like]: `%${query.search}%` } }
         ]
       });
     }
-
-    builder.where({
-      organisationId: {
-        [Op.in]: user.organisations.map(org => org.uuid)
-      }
-    });
-
-    return { data: await builder.execute(), paginationTotal: await builder.paginationTotal(), pageNumber };
-  }
-
-  async getAdminProjectPitches(params: EntityQueryDto) {
-    const pageNumber = params.page ? params.page.number : 1;
-    const pageSize = params.page ? params.page.size : MAX_PAGE_SIZE;
-    const organisationAssociation: Includeable = {
-      association: "organisation",
-      attributes: ["uuid", "name"]
-    };
-    const builder = new PaginatedQueryBuilder(ProjectPitch, pageSize, [organisationAssociation]);
-    if (pageNumber > 1) {
-      builder.pageNumber(pageNumber);
-    }
-
-    if (params.search) {
-      builder.where({
-        [Op.or]: [
-          { projectName: { [Op.like]: `%${params.search}%` } },
-          { "$organisation.name$": { [Op.like]: `%${params.search}%` } }
-        ]
+    if (query.filter) {
+      Object.keys(query.filter).forEach(key => {
+        if (!["restoration_intervention_types", "project_country"].includes(key)) {
+          throw new BadRequestException(`Invalid filter key: ${key}`);
+        }
+        const value = query.filter[key];
+        builder.where({
+          [key]: { [Op.like]: `%${value}%` }
+        });
       });
     }
+    if (query.sort != null) {
+      if (
+        [
+          "id",
+          "organisation_id",
+          "project_name",
+          "project_objectives",
+          "project_country",
+          "project_county_district",
+          "restoration_intervention_types",
+          "total_hectares",
+          "total_trees",
+          "capacity_building_needs",
+          "created_at",
+          "updated_at",
+          "deleted_at"
```

Comment:
These sort fields should all be camel case. Also, why is `deleted_at` here? There's no way for the FE to receive deleted entities typically, so sorting on that field makes no sense. Also `id`? That should be the default sort, and isn't typically needed as an explicit sort from the FE - it's a bit weird for the FE to request sorting on a field that it doesn't receive. In general, please make sure these are all actually needed in the places pitches are displayed. There's usually only something like 2-5 fields that we really need to sort an index on.